contracts/BC_fusion/StakeHub.sol

```diff
@@ -712,6 +721,31 @@ contract StakeHub is System, Initializable {
         return _paused;
     }
 
+    /**
+     * @notice the returned reward may be not accurate, it's been truncated
+     * @return the validator's reward and apy of the day(with 2 decimals)
+     */
+    function getValidatorRewardInfo(
+        address operatorAddress,
+        uint256 dayIndex
+    ) external view returns (uint256, uint256) {
+        uint256 info = _rewardInfo[operatorAddress][dayIndex];
+        uint256 mask = (1 << 120) - 1;
```

Comment:
I dont mind to store one more uint256 instead of so many tricky calculation here, because break block is very rare.